test/libsolidity/util/BytesUtils.cpp

```diff
@@ -219,7 +219,7 @@ string BytesUtils::formatString(bytes const& _bytes, size_t _cutOff)
 				if (isprint(v))
 					os << v;
 				else
-					os << "\\x" << toHex(v);
+					os << "\\x" << toHex(v, HexCase::Lower);
```

Comment:
I think this is the only other place where we call `toHex()` with a single char argument (BTW, because of `auto`, I'm not 100% sure which type is `v` here, I'd have to check how `bytes` is defined).

I did not manage to trigger this in tests but I changed it too just in case.